pkgs/development/libraries/cjson/default.nix

```
@@ -0,0 +1,23 @@
+{stdenv, fetchFromGitHub, cmake}:
```

You need to include both `stdenv` and `lib`. Add space after `{` and before `}`. 

```suggestion
{ stdenv, lib, fetchFromGitHub, cmake }:
```